ideascube/monitoring/tests/test_views.py

```diff
@@ -325,6 +358,48 @@ def test_cant_create_inventoryspecimen_by_barcode_twice(staffapp):
                                             specimen=specimen).count() == 1
 
 
+def test_staff_can_create_inventoryspecimen_by_serial(staffapp):
```

Comment:
> The original need is to be able to make inventory using serial to not have to print a barcode.

Not have to print it, yes.

But then it still might be printed at a later date, because somebody had a bit too much time.

I think these mixed tests would be very useful.